packages/components/tests/dummy/app/templates/components/pagination-bar.hbs

```diff
@@ -65,33 +206,236 @@
 <section>
   <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
 
-  {{! ADD YOUR CONTENT HERE }}
-
+  <!-- <Doc::WcagList
+    @criteriaList={{array
+      "1.3.1"
+      "1.3.2"
+      "1.4.1"
+      "1.4.3"
+      "1.4.4"
+      "1.4.10"
+      "1.4.11"
+      "1.4.12"
+      "2.1.1"
+      "2.1.2"
+      "2.4.3"
+      "2.4.4"
+      "2.4.6"
+      "2.4.7"
+      "2.5.3"
+      "3.2.1"
+      "4.1.2"
+    }}
+  /> -->
 </section>
 
 <section data-test-percy>
   <h3 class="dummy-h3" id="showcase">
     <a href="#showcase" class="dummy-link-section">§</a>
     Showcase
   </h3>
+  <button id="dummy-toggle-highlight" type="button" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide" "Show"}}
+    layout highlight
+  </button>
+
+  <div class="{{if this.showHighlight 'dummy-pagination-layout-highlight'}}">
+
+    <h4 class="dummy-h4">Base elements</h4>
+
+    <h5 class="dummy-h5">TotalCount</h5>
+
+    <span class="dummy-text-small">with <code class="dummy-code">@totalItems</code></span>
+    <Hds::PaginationBar::TotalCount @itemsPerPage={{10}} @currentPage={{1}} @totalItems={{103}} />
+
+    <!--
+    <span class="dummy-text-small">without <code class="dummy-code">@totalItems</code></span>
+    <Hds::PaginationBar::TotalCount @itemsPerPage={{10}} @currentPage={{1}} />
+    -->
```

Comment:
@didoo I commented these examples out because `@totalItems` is required. Let me know if it's ok to delete these or if there are other refactors you think should be made.